package.json

```diff
@@ -31,7 +31,10 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "start:emulators": "cross-env REACT_APP_USE_EMULATOR=true concurrently \"npm run start-app-emulated\" \"npm run start-firebase-emulated\"",
+    "start-app-emulated": "react-scripts start",
```

Comment:
Since this is equivalent to line 31 above, it would probably make more sense to just use `npm start` directly.

Reply:
yes, before i was doing REACT_APP_USE_EMULATOR=true inside start-app-emulated but then i moved it so now it's redundant